plugin/evm/config.go

```diff
@@ -173,6 +173,11 @@ type Config struct {
 	RegossipFrequency         Duration         `json:"regossip-frequency"`
 	PriorityRegossipAddresses []common.Address `json:"priority-regossip-addresses"`
 
+	// Order Gossip Settings
+	OrderGossipNumValidators    int `json:"order-gossip-num-validators"`
```

Comment:
should we add defaults in code, like other config fields?

Reply:
done